clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -168,6 +168,19 @@ New checks
   extracted from an optional-like type and then used to create a new instance
   of the same optional-like type.
 
+- New :doc:`bugprone-unused-local-non-trivial-variable
+  <clang-tidy/checks/bugprone/unused-local-non-trivial-variable>` check.
+
+  Warns when a local non trivial variable is unused within a function. By
```

Comment:
Release Notes should contain single statement.

Reply:
Done.